src/registry/routes/list-components.js

```diff
@@ -47,18 +47,37 @@ module.exports = function(repository){
             return componentInfo.name;
           });
 
-          return res.render('list-components', _.extend(baseResponse, {
+          var deprecated = 0,
+              experimental = 0;
+
+          var vm = _.extend(baseResponse, {
             availableDependencies: res.conf.dependencies,
             availablePlugins: res.conf.plugins,
             components: componentsInfo,
             componentsReleases: componentsReleases,
             componentsList: _.map(componentsInfo, function(component){ 
-              return {
+              var mapped = {
                 name: component.name,
                 state: (!!component.oc && !!component.oc.state) ? component.oc.state : ''
-              }; 
-            })
-          }));
+              };
+
+              if(mapped.state === 'deprecated'){
+                deprecated++;
+              } else if(mapped.state === 'experimental'){
+                experimental++;
+              }
+
```

Comment:
I removed this actually in favour of a more generic counts object that handles counts for any state. This should be also more flexible in case we add extra states in the future